vmdb/app/controllers/api_controller/action.rb

```diff
@@ -0,0 +1,33 @@
+class ApiController
+  module Action
+    private
+
+    def api_action(type, id, options = {})
+      cspec = collection_config[type]
+      klass = cspec[:klass].constantize
+
+      result = yield(klass) if block_given?
+
+      add_href_to_result(result, type, id) if options[:add_href]
```

Comment:
good point :) Upcoming PR's that don't need it, but it's more of the 20% exception, will change to skip_href.